lib/addons/opa-gatekeeper/index.ts

```diff
@@ -0,0 +1,69 @@
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+/**
+ * Properties available to configure opa gatekeeper
+ */
+export interface OpaGatekeeperAddOnProps {
+    /**
+     * Namespace where OPA Gatekeeper will be installed
+     * @default true
+     */
+    createNamespace?: boolean;
+    /** Default namespace
+     * @default gatekeeper-system
+     */
+    namespace?: string;
+    /**
+     * Helm chart version to use to install.
+     * @default 3.7.0-beta.1
+     */
+    version?: string;
+    /**
+     * 	Whether or not Helm should wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment, StatefulSet, or ReplicaSet are in a ready state before marking the release as successful.
+     * @default true
+     */
+    wait?: boolean;
```

Comment:
This option is not leveraged for the helm chart. That means that if customers pass this value, it will have no impact on the provisioned add-on.

Reply:
Removed - not necessary for the helm chart